docs/OMS-PostgreSQL-Instructions.md

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
Add a step here about acquiring the workspace ID for the workspace that they want the logs to be sent to.